### DISCUSSION TITLE
QQ最新版V9.9.3无法打补丁

### DISCUSSION THREAD

**Discussion by luyingwei80** [huiyadanli/RevokeMsgPatcher#637] [General]
在最新的QQ新版V9.9.3中已无法正常打防撤回补丁了。
![image](https://github.com/huiyadanli/RevokeMsgPatcher/assets/67801546/c5775ef3-8e6f-46b6-82d6-d55694c823f7)

**Comment by bananaguys** [comment]
我也是怎么办，QQ 9.9.7
